include/vkd3d_shader.h

```diff
@@ -370,6 +370,8 @@ enum vkd3d_shader_target_extension
     VKD3D_SHADER_TARGET_EXTENSION_SUPPORT_FP64_DENORM_PRESERVE,
     VKD3D_SHADER_TARGET_EXTENSION_SUPPORT_SUBGROUP_PARTITIONED_NV,
     VKD3D_SHADER_TARGET_EXTENSION_COMPUTE_SHADER_DERIVATIVES_NV,
+    VKD3D_SHADER_TARGET_EXTENSION_MAXIMAL_RECONVERGENCE,
```

Comment:
Quad control extension depends on maximal reconvergence, so we should merge the extension to avoid the awkward inner loop.